conans/test/functional/command/json_output_test.py

```diff
@@ -125,6 +125,7 @@ def build(self):
         self.assertIn("CC/1.0@private_user/channel: Error in build() method, line 36",
                       my_json["installed"][0]["packages"][0]["error"]["description"])
 
+    @unittest.expectedFailure  # TODO: Need CLI in the cross-building feature to fix it
```

Comment:
CLI?

Reply:
CLI ~ Command line interface.

This PR has the command line interface already merged, so it should be _easy_ to make the needed changes to make it pass.